pynta/tasks.py

```diff
@@ -227,7 +227,7 @@ def run_task(self, fw_spec):
             try:
                 sp.calc.close()
             except Exception as e:
-                if self["software"] == "Espresso":
+                if self["software"] == "Espresso" or self["software"] == "DP" or self["software"] == "Vasp":
```

Comment:
This was for handling a very specific error when we were initially trying to run Espresso through sockets to make it faster...eventually we decided to not do that because it was causing too many problems...do Vasp and DP even support sockets? and have the same issue?

Reply:
Vasp does support sockets. I don't think sockets would be necessary for DP. 

Let's discuss what problems you had with Espresso with sockets, maybe we can use them for vasp. I can run some tests